nodejs-18.yaml

```diff
@@ -10,6 +10,9 @@ package:
         - "*"
       attestation: TODO
       license: MIT
+  dependencies:
+    provides:
+      - nodejs=18.14.999
```

Comment:
what does `999` mean here? it will satifies any package require `18.14.x` going forward ?

Reply:
Not entirely sure honestly! I copied this from go: https://github.com/wolfi-dev/os/blob/main/go-1.20.yaml#L15

Reply:
This means "when someone asks for `nodejs`, pretend this package provides `nodejs 18.14.999` -- this will always make it the highest-versioned `nodejs` and select it unless there's a `nodejs=19`+.

TBH I don't really love this convention, and I wish we'd just have it provide `nodejs=18.14.0` explicitly, and just update it when we update this package's version. It seems to only use `999` as a convenience to mean "the highest patch version possible". It does prevent us from forgetting to bump it when we update the package though.